internal/compiler/typeregister.rs

```diff
@@ -580,7 +581,7 @@ impl TypeRegister {
         register.elements.remove("DropArea").unwrap();
         register.types.remove("DropEvent").unwrap(); // Also removed in xtask/src/slintdocs.rs
 
-        register.elements.remove("MarkdownText").unwrap();
+        register.elements.remove("StyledText").unwrap();
```

Comment:
If we want to keep the feature internal, we also need to remove the "styled-text" from register.types